src/views/user/official-contact-email-verification.ejs

```diff
@@ -24,25 +24,30 @@
                                 Insérer le code reçu
                             </label>
                             <input class="fr-input" required
-                                   type="text" id="official_contact_email_verification_token"
-                                   name="official_contact_email_verification_token"
-                                   autocomplete="off">
+                                    type="text" id="official_contact_email_verification_token"
```

Comment:
nitpick: this kind of changes should not happen. You should consider configuring your IDE not to make unintended changes.